bundles/org.openhab.binding.nikohomecontrol/src/main/java/org/openhab/binding/nikohomecontrol/internal/handler/NikoHomeControlActionHandler.java

```diff
@@ -103,25 +79,24 @@ private void handleCommandSelection(ChannelUID channelUID, Command command) {
             case CHANNEL_BUTTON:
             case CHANNEL_SWITCH:
                 handleSwitchCommand(command);
-                updateStatus(ThingStatus.ONLINE);
                 break;
             case CHANNEL_BRIGHTNESS:
                 handleBrightnessCommand(command);
-                updateStatus(ThingStatus.ONLINE);
                 break;
             case CHANNEL_ROLLERSHUTTER:
                 handleRollershutterCommand(command);
-                updateStatus(ThingStatus.ONLINE);
                 break;
             default:
                 logger.debug("unexpected command for channel {}", channelUID.getId());
         }
+
+        updateStatus(ThingStatus.ONLINE);
```

Comment:
Why are you updating the thing status when a command is sent to a channel ? I don't understand the logic.

Same comment for all classes implementing handleCommandSelection

Reply:
Not needed.